package.json

```diff
@@ -158,7 +168,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run lint",
+      "pre-push": "commitlint --from origin/develop --to HEAD --verbose",
```

Comment:
has code linting been removed?

Reply:
Code linting has been moved to pre-commit in an earlier PR. Linting errors should be caught then and during CI, so that we don't have to wait for the pre-push hook to lint the entire codebase before pushing

Reply:
Sorry to be nosy, but propose using `commit-msg` hook instead of `pre-push`.

Also suggest using `HUSKY_GIT_PARAMS`, which automatically retrieves the git params that invoked the husky hook.
```
"husky": {
  "hooks": {
    "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
  }
},
```